lib/active_merchant/billing/gateways/visanet_peru.rb

```diff
@@ -0,0 +1,239 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class VisanetPeruGateway < Gateway
+      self.display_name = "VisaNet Peru Gateway"
+      self.homepage_url = "http://www.visanet.com.pe"
+
+      self.test_url = "https://devapi.vnforapps.com/api.tokenization/api/v2/merchant"
+      self.live_url = "https://api.vnforapps.com/api.tokenization/api/v2/merchant"
+
+      self.supported_countries = ["US", "PE"]
+      self.default_currency = "PEN"
+      self.money_format = :cents
+      self.supported_cardtypes = [:visa, :master, :american_express, :discover]
+
+      def initialize(options={})
+        requires!(options, :login, :password)
```

Comment:
We should use credential names that directly reflect what the gateway calls them, i.e. `:access_key_id` and `:secret_access_key` for VisaNet.